Find a functionally equivalent code:android.provider.DocumentsContract.getDocumentThumbnail:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * {@hide}
 */

@@ -1,6 +1,6 @@
 {
     final Bundle openOpts = new Bundle();
-    openOpts.putParcelable(DocumentsContract.EXTRA_THUMBNAIL_SIZE, size);
+    openOpts.putParcelable(ContentResolver.EXTRA_SIZE, size);
     AssetFileDescriptor afd = null;
     Bitmap bitmap = null;
     try {
@@ -11,17 +11,17 @@
         // optimal decode path; otherwise fall back to buffering.
         BufferedInputStream is = null;
         try {
-            Libcore.os.lseek(fd, offset, SEEK_SET);
+            Os.lseek(fd, offset, SEEK_SET);
         } catch (ErrnoException e) {
             is = new BufferedInputStream(new FileInputStream(fd), THUMBNAIL_BUFFER_SIZE);
             is.mark(THUMBNAIL_BUFFER_SIZE);
@@ -34,40 +34,40 @@
             bitmap = BitmapFactory.decodeStream(is, null, opts);
         } else {
             try {
-                Libcore.os.lseek(fd, offset, SEEK_SET);
+                Os.lseek(fd, offset, SEEK_SET);
             } catch (ErrnoException e) {
                 e.rethrowAsIOException();
             }

